packages/lesswrong/lib/collections/votes/schema.ts

```diff
@@ -179,6 +179,14 @@ const schema: SchemaType<DbVote> = {
     type: Boolean,
     canRead: ['guests'],
     ...schemaDefaultValue(false)
+  },
+
+  // Whether to notify users of the karma changes from this vote. This is set to false for votes that are
+  // nullified by mod actions
+  notify: {
+    type: Boolean,
```

Comment:
I'm generally a fan of making the default values of booleans false. So in this case I would make this a boolean for silenceNotification. Maybe this is due to my terrible javascript-addled brain, but it feels safer. It also feels easier (for my simply addled brain) to reason about flags being off by default.